Jenkinsfile.nightly-build

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import java.text.SimpleDateFormat
+
+properties([pipelineTriggers([cron('15 2 * * *')])])
+
+def dateFormat = new SimpleDateFormat("yyyyMMdd")
+def date = new Date()
+def timestamp = dateFormat.format(date)
+
+pipeline {
+    agent { 
+        node { 
+            label 'nightly-build'
+            // Forcing a short path workspace to prevent issues with arguments lentgh
+            customWorkspace "workspace/${URLDecoder.decode(JOB_NAME)}/${BUILD_NUMBER}"
+        }
+    }
+    stages {
+        stage('Generate release') {
+            steps {
+                sh """
+                npm install
+                node ./node_modules/.bin/grunt release
+                jq -r '.version' config.json > version
+                """
+                script {
+                    env.LISK_VERSION = readFile("version").trim()
+                }
+            }
+        }
+        stage('Prepare build environment') {
+            steps {
+                dir('lisk-build') {
+                    git url: "https://github.com/LiskHQ/lisk-build.git", branch: "1.0.0"
+                }
+                sh "cp release/lisk-${env.LISK_VERSION}.tgz lisk-build/src"
+            }
+        }
+        stage('Build release') {
+            steps {
+                dir('lisk-build') {
+                    sh """
+                    bash build.sh -n dev -v ${env.LISK_VERSION}
+                    mv release/lisk-${env.LISK_VERSION}-Linux-x86_64.tar.gz release/lisk-${env.LISK_VERSION}-${timestamp}-Linux-x86_64.tar.gz
+                    sha256sum release/lisk-${env.LISK_VERSION}-${timestamp}-Linux-x86_64.tar.gz > release/lisk-${env.LISK_VERSION}-${timestamp}-Linux-x86_64.tar.gz.SHA256
```

Comment:
This is already been don in `lisk-build` so no need it here in jenkins. 

https://github.com/LiskHQ/lisk-build/blob/4efa4655639eee36aa8e8301671289844e385940/build.sh#L227-L229

Reply:
This part appends the timestamp, which is not the case in `lisk-build`

Reply:
That's what I am suggesting to add this to `lisk-build` to be used everywhere.

Reply:
@Nazgolze Is this something we could add to `lisk-build` ?